packages/edit-post/src/components/sidebar/post-link/style.scss

```
@@ -0,0 +1,22 @@
+.edit-post-post-link__link-prefix,
+.edit-post-post-link__link-suffix {
+	font-weight: 600;
+}
+
+.edit-post-post-link__link-post-name {
+	font-weight: 800;
+}
+
+.edit-post-post-link__preview-label {
+	margin: 0;
+}
+
+.edit-post-post-link__link {
+	word-break: break-word;
```

Not supported in Firefox, see https://caniuse.com/#feat=word-break notes
```suggestion
	word-wrap: break-word;
```